terminus-ui/table/src/table.component.scss

```diff
@@ -1,51 +1,129 @@
 @import './../../scss/helpers/color';
+@import './../../scss/helpers/cursors';
 @import './../../scss/helpers/layout';
 @import './../../scss/helpers/reset';
-@import './../../scss/helpers/scrollbars';
 @import './../../scss/helpers/spacing';
+@import './../../scss/helpers/scrollbars';
 @import './../../scss/helpers/typography';
 
-$ts-header-row-height: 56px;
-$ts-row-height: 48px;
-$ts-row-horizontal-padding: 24px;
-$ts-row-hover: rgba(color(utility, xlight), .5);
-
-
 
 //
 // @component
 //  Table
 // @description
 //  A table component
-//
 .ts-table {
+  --table-bg: #{color(pure)};
+  --header-bg: #{color(utility, xlight)};
+  --header-text-color: #{color(utility, dark)};
+  --border-color: #{color(utility, light)};
+  $primary: color(primary);
+  --row-bg--hover: #{desaturate(lighten($primary, 66%), 70%)};
+  // NOTE: This must be above 40 as that is the number of header cell z-indexes set
+  --header-cell-sticky-z-index: 50;
+  --row-cell-sticky-z-index: 3;
+  --z-index-negative: -1;
+  --z-index-base-context: 1;
```

Comment:
Just a heads up, I've started using CSS variables more rather than only SCSS variables. I think this will help us skin items easier in the future (ie rather than generating separate styles for primary button, accent button, etc, we can just change the CSS variable).